packages/db/lib/gen-migration.mjs

```diff
@@ -50,7 +49,8 @@ function getNextMigrationId (migrationFileNames) {
 
 async function generateMigrationFile (migrationDir, id) {
   const newMigrationFilePath = path.join(migrationDir, id)
-  await fs.promises.appendFile(newMigrationFilePath, '')
+  const fh = await open(newMigrationFilePath, 'w+')
```

Comment:
You don't need this. Just write into file.

Reply:
Thanks, done.